app/views/system_admin/users/show.html.erb

```diff
@@ -1,69 +1,100 @@
 <%= render PageTitle::View.new(text: @user.name) %>
 
 <%= content_for(:breadcrumbs) do %>
-  <%= render GovukComponent::BackLinkComponent.new(
-    text: t("back"),
-    href: users_path,
-    ) %>
+  <%= render GovukComponent::BackLinkComponent.new(text: t(:back), href: users_path) %>
 <% end %>
 
 <h1 class="govuk-heading-l"><%= @user.name %></h1>
 
 <dl class="govuk-summary-list">
   <div class="govuk-summary-list__row">
-    <dt class="govuk-summary-list__key">
-      First name
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <%= @user.first_name %>
-    </dd>
+    <dt class="govuk-summary-list__key">First name</dt>
+    <dd class="govuk-summary-list__value"><%= @user.first_name %></dd>
   </div>
   <div class="govuk-summary-list__row">
-    <dt class="govuk-summary-list__key">
-      Last name
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <%= @user.last_name %>
-    </dd>
+    <dt class="govuk-summary-list__key">Last name</dt>
+    <dd class="govuk-summary-list__value"><%= @user.last_name %></dd>
   </div>
   <div class="govuk-summary-list__row">
-    <dt class="govuk-summary-list__key">
-      Email
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <%= @user.email %>
-    </dd>
+    <dt class="govuk-summary-list__key">Email</dt>
+    <dd class="govuk-summary-list__value"><%= @user.email %></dd>
   </div>
-  <div class="govuk-summary-list__row lead-school-row">
-    <dt class="govuk-summary-list__key">
-      Lead schools
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <% if @user.lead_schools.any? %>
-        <div class="govuk-!-margin-bottom-3">
-          <% @user.lead_schools.each do |lead_school| %>
-            <%= govuk_link_to("#{lead_school.name} - #{lead_school.urn}", lead_school_path(lead_school), class: "govuk-!-display-block") %>
-          <% end %>
-        </div>
-      <% end %>
-      <%= govuk_link_to("Add lead school", new_user_lead_school_path(@user), class: "govuk-!-display-block add-lead-school-to-user") %>
-    </dd>
-  </div>
-  <div class="govuk-summary-list__row provider-row">
-    <dt class="govuk-summary-list__key">
-      Providers
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <% if @user.providers.any? %>
-        <div class="govuk-!-margin-bottom-3">
+</dl>
+
+<h2 class="govuk-heading-m">Providers</h2>
+<% if @user.providers.any? %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <table id="providers" class="govuk-table">
+        <caption class="govuk-visually-hidden govuk-table__caption govuk-table__caption--m">Providers</caption>
```

Comment:
```suggestion
```

I don't think these captions are needed as we have headings that describe the content.

Reply:
I did try to remove it, but SonarCloud complained that a table should have a caption. It seems all our tables have a caption.